host/3.0/buster/amd64/powershell/powershell7.Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Build the runtime from source
+ARG HOST_VERSION=3.0.14358
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest false
```

Comment:
Nitpick: the original script (powershell6.Docker) uses the `=` syntax, and the syntax without `=` is discouraged now (AFAIK). Should we fix this?